mailpoet/assets/js/src/landingpage/header.tsx

```diff
@@ -0,0 +1,28 @@
+import { MailPoet } from 'mailpoet';
+import { Button } from 'common';
+import { Heading } from 'common/typography/heading/heading';
+
+function Header(): JSX.Element {
+  return (
+    <div className="mailpoet-content-center">
+      <Heading level={1}>
+        {' '}
+        {MailPoet.I18n.t('betterEmailWithoutLeavingWordPress')}{' '}
+      </Heading>
+      <Heading level={3}>
+        {' '}
+        {MailPoet.I18n.t('startingOutOrEstablished')}{' '}
+      </Heading>
+      <Button
+        onClick={() => {
+          window.location.href = window.mailpoet_welcome_wizard_url;
```

Comment:
I think defining this as a named function and using it here and in `Footer` component would be a good idea, some of the values would be: 
1) The function is not re-defined in twice and again in each re-render
2) A named function can give us more information about what happens without needing to actually read the code
What do you think about this suggestion?

Reply:
Noted